aws/CreateFeedback.md

```diff
@@ -0,0 +1,57 @@
+# Aiven Docs submitting feedback
+
+Feedback are submitted to a PostgreSQL database maintained by Aiven. Production URI can be found in Devportal 1Password.
+
+## Data
+
+Currently the feedback table has 4 columns:
+
+- **referrer** (string): Page URL where the feedback is submitted. Automatically filled
+- **vote** (string): "up" / "down". Either a upvote or downvote. Filled by user when clicking corresponding buttons.
+- **message** (string): Feedback field. Filled by user.
+- **created_at** (string): Timestamp of the submitted feedback. Automatically filled.
+
+The JS script in `feedback-form.html` would pick up `referrer`, `vote` and `message`by itself while `created_at`is assigned by the lambda upon creating the feedback in the database.
+
+## Testing locally
+
+### One time testing
+
+Following the instructions [here](link) one should have a `event.CreateFeedbackFunction.local.json` after running `npm run generate-event POST CreateFeedbackFunction`. It looks like this
```

Comment:
should there be a link here?

Reply:
Will fix

Reply:
@lornajane Added the switch plan to this PR description